src/game/game/game.c

```diff
@@ -68,8 +77,8 @@ static int32_t Game_Control(int32_t nframes, GAMEFLOW_LEVEL_TYPE level_type)
             }
         }
 
-        if ((g_InputDB.option || g_Input.save || g_Input.load
-             || g_OverlayFlag <= 0)
+        if (((g_Config.enable_buffering ? g_Input : g_InputDB).option
+             || g_Input.save || g_Input.load || g_OverlayFlag <= 0)
```

Comment:
Holy cow, what's this about?

Reply:
Do you think this is a better approach? It's maybe a bit more readable.
```
        if (((g_Config.enable_buffering ? g_Input.option : g_InputDB.option)
             || g_Input.save || g_Input.load || g_OverlayFlag <= 0)
```

Reply:
I mostly mean the complexity of this condition

Reply:
Yes, it's quite lengthy. Should we move it to a function?

Reply:
No, what we need is a rewrite of the inventory code :D